src/js/main.js

```diff
@@ -19,8 +20,14 @@ Vue.use(VueRouter)
 const router = new VueRouter({ routes })
 
 // Store
+const vuexLocalStorage = new VuexPersist({
+  key: 'vuex',
+  storage: window.localStorage,
+})
```

Comment:
@Bronson-Brown-deVost : note that I moved `VuexPersist` up to the top level. The reason for this is that we should be able to create stores for testing purposes without needing to have a dependency on localStorage.